Procfile

```diff
@@ -0,0 +1 @@
+web: bundle exec puma -C config/puma.rb
```

Comment:
Is this change related to view inspections? We might want to move this into a separate PR so that when folk review changes and PRs in the future the changes are documented accurately.